yaml/stringify.ts

```diff
@@ -28,7 +28,7 @@ export type StringifyOptions = {
   /** Each tag may have own set of styles.	- "tag" => "style" map. */
   styles?: Record<string, "lowercase" | "uppercase" | "camelcase" | "decimal">;
   /** Name of the schema to use. */
-  schema?: "core" | "default" | "failsafe" | "json" | "extended";
+  schema?: keyof typeof SCHEMA_MAP;
```

Comment:
A public API can't reference private type to be able to publish to jsr (that causes lint error). I think this need to be literal type.
```suggestion
  schema?: "core" | "default" | "failsafe" | "json" | "extended";
```